howitworks.md

```diff
@@ -6,6 +6,18 @@ top_graphic: 3
 redirect_from: "/howitworks"
 ---
 
+# What is Let's Encrypt?
+
+Let's Encrypt allows you to secure your website by concealing the data
```

Comment:
I would instead say "improve the security of your website." And "concealing the data" sounds a bit negative (even though it's accurate, and it's good to conceal browsing data :-)). At https://www.eff.org/encrypt-the-web, I tried to explain the specific things HTTPS protects against, eavesdropping and content hijacking. Maybe incorporate those here? I would also suggest mentioning HTTPS (HTTP plus SSL or TLS) in the first sentence.